plugins/embark-assistant/defs.h

```diff
@@ -68,24 +68,29 @@ namespace embark_assist {
             Woodland,
             Heavily_Forested
         };
-
-        struct mid_level_tile {
+        
+        // only contains those attributes that are being handled during incursion processing
+        struct mid_level_tile_incursion_base {
```

Comment:
Yeah, I wasn't able to come up with a better name either - if there were multiple derived structs, perhaps `_common` instead of `_base` could work (but there aren't)

Reply:
Nonetheless: Thank you for putting some thought into it!